.pylintrc

```diff
@@ -87,7 +86,7 @@ output-format=text
 # command line instead of printing them on stdout. Reports (if any) will be
 # written in a file name "pylint_global.[txt|html]". This option is deprecated
 # and it will be removed in Pylint 2.0.
-files-output=no
+#files-output=no
```

Comment:
How comes this changed?

Reply:
files-ouput was removed in PyLint [1.7](https://pylint.pycqa.org/en/latest/whatsnew/1/1.7/summary.html#removed-changes)